packages/issuer-sdk-js/README.md

```diff
@@ -9,13 +9,150 @@ Get [our NPM package](https://www.npmjs.com/package/@idos-network/idos-sdk) and
 pnpm add @idos-network/idos-issuer-sdk-js
 ```
 
+
+## Setting up
+
+Create an issuer config with your secret key. This config will be used to interact with the idOS.
+```js
+// issuer-config.js
+import { createIssuerConfig } from "@idos-network/idos-issuer-sdk-js";
+import { Wallet } from "ethers";
+
+const wallet = new Wallet("YOUR_PRIVATE_KEY");
+
+const issuerConfig = await createIssuerConfig({
+  nodeUrl: "https://nodes.idos.network/", // or nodes.playground.idos.network
+  encryptionSecret: "YOUR_SECRET_KEY",
+  signer: wallet
+});
+```
+### Creating a human profile.
+
+In order to create a human profile in idOS, you need to have a wallet associated with the human and a public key for the human.
+Passing ids are optional and will be auto-generated if not provided.
+
+```js
+// createHumanDemo.js
+
+import { createHuman } from "@idos-network/idos-issuer-sdk-js";
+import issuerConfig from "./issuer-config.js";
+
+/**
+ * Represents a human profile in the idOS system.
+ * 
+ * @typedef {Object} Human
+ * @property {string} current_public_key - The public key derived from the user's keypair, used to encrypt credentials content.
+ */
```

Comment:
Let's not do jsdocs in the middle of prose. If you want to "talk outside code", close the code block, and just talk. :P